contribution_evaluation.py

```
@@ -0,0 +1,304 @@
+
+
+# get content in file
+def get_file_content(owner, repo, file_path, sha):
+    url = f"{API_BASE_URL}/repos/{owner}/{repo}/contents/{file_path}?ref={sha}"
+    headers = {
+        "Authorization": f"token {TOKEN}",
+        "Accept": "application/vnd.github.raw+json"  # use this 'Accept' when you get origin file content
+    }
+    return send_request(url, headers)
+# get file content (origin content)
+def get_file_content(owner, repo, file_path, sha):
```

Ruff (0.8.2)

Redefinition of unused `get_file_content` from line 76

(F811)